include/openthread/srp_server.h

```diff
@@ -399,6 +430,42 @@ const char *otSrpServerHostGetFullName(const otSrpServerHost *aHost);
  */
 const otIp6Address *otSrpServerHostGetAddresses(const otSrpServerHost *aHost, uint8_t *aAddressesNum);
 
+/**
+ * This function returns the lease time in seconds of a given host.
+ *
+ * @param[in]  aHost  A pointer to the SRP server host.
+ *
+ * @returns  An integer that represents the lease time in seconds of the given host.
+ */
+uint32_t otSrpServerHostGetLease(const otSrpServerHost *aHost);
+
+/**
+ * This function returns the key lease time in seconds of a given host.
+ *
+ * @param[in]  aHost  A pointer to the SRP server host.
+ *
+ * @returns  An integer that represents the key lease time in seconds of the given host.
+ */
+uint32_t otSrpServerHostGetKeyLease(const otSrpServerHost *aHost);
+
+/**
+ * This function returns the expire time in milliseconds of a given host.
+ *
+ * @param[in]  aHost  A pointer to the SRP server host.
+ *
+ * @returns  An integer that represents the expire time in milliseconds of the given host.
+ */
+uint32_t otSrpServerHostGetExpireTime(const otSrpServerHost *aHost);
```

Comment:
What we return here seems to be the `TimeMille` instance as `uint32_t`? This is  basically tied to the notion of time from `TimerMille::GetNow()` or `otPlatAlarmMilliGetNow()`. This time value is somewhat internal to OT stack itself and the platform, and not something that users of OT API need to be aware of (we don't expose `GetNow()` as a public API which I think is a good model).

My suggestion for these public APIs is to change them such that we return "the remaining number of milliseconds till entry expires" (counting the msec from the time the API is called). This makes this definition simple and not tied to notion of the internal `GetNow()` time.

Reply:
Changing the API to "the remaining number of milliseconds till entry expires" sounds good to me. The only thing I'm not sure is, will there be any performance concerns? It seems that every API call will trigger a `GetNow()` call.

Reply:
+1 on @abtink 's proposal. I don't believe it's a concern to call `GetNow()` everytime.

Reply:
Functions updated. Thanks for the suggestion!